### PR TITLE
test(spanner): skip primary key tests for numeric

### PR DESCRIPTION
**Pull request by jiren**
- skip numeric primary acceptance tests due to numeric primary not supported.

### CODE REVIEW

File: google-cloud-spanner/acceptance/spanner_helper.rb
Comment (hengfengli) [inline google-cloud-spanner/acceptance/spanner_helper.rb:363]: Is if safe to do this operation? Do we have a database name convention so that we only delete databases related to this test?

Comment (hengfengli) [reply google-cloud-spanner/acceptance/spanner_helper.rb:363]: Or only delete `stale` databases, e.g., was created a day ago.

Comment (jiren) [reply google-cloud-spanner/acceptance/spanner_helper.rb:363]: Yes, updated code.

File: google-cloud-spanner/acceptance/spanner_helper.rb
Comment (hengfengli) [inline google-cloud-spanner/acceptance/spanner_helper.rb:44]: Please also only delete stale databases that were created 1 day ago. I am worried that if multiple tests running at the same time will affect each other.

**Comment by hengfengli** [comment]
@jiren Can you please make `test database cleanup` as a separate PR? So that, we can first skip numeric pk test and let the acceptance passing. I can try to manually delete the stale databases to clean the way. Then, after acceptance test is okay, we can merge your PR on adding create_time for databases, and next would be the `test database cleanup`. So the order: 

1. Skip pk tests for numeric
2. Add create_time for databases
3. Clean stale test databases

**Comment by jiren** [comment]
> @jiren Can you please make `test database cleanup` as a separate PR? So that, we can first skip numeric pk test and let the acceptance passing. I can try to manually delete the stale databases to clean the way. Then, after acceptance test is okay, we can merge your PR on adding create_time for databases, and next would be the `test database cleanup`. So the order:
> 
> 1. Skip pk tests for numeric
> 2. Add create_time for databases
> 3. Clean stale test databases

ok, I will create separate PRs.

**Comment by jiren** [comment]
@hengfengli Removed instances cleanup.

**Comment by hengfengli** [comment]
Hi @dazuma, we got the following error due to the limit of max number on restoring operations: 

```
  1) Error:
Spanner Database Backup::spanner#test_0001_creates, get, updates, restore and delete a database backup:
Google::Cloud::FailedPreconditionError: 9:Cannot restore database (projects/helical-zone-771/instances/google-cloud-ruby-tests/databases/restore-gcruby-210521-faa30f2b) because the maximum number of pending restores (1) for the instance (projects/helical-zone-771/instances/google-cloud-ruby-tests) has been reached. Please retry the operation once the pending restores complete.. debug_error_string:{"created":"@1621622675.503349447","description":"Error received from peer ipv4:74.125.195.95:443","file":"src/core/lib/surface/call.cc","file_line":1066,"grpc_message":"Cannot restore database (projects/helical-zone-771/instances/google-cloud-ruby-tests/databases/restore-gcruby-210521-faa30f2b) because the maximum number of pending restores (1) for the instance (projects/helical-zone-771/instances/google-cloud-ruby-tests) has been reached. Please retry the operation once the pending restores complete.","grpc_status":9}
    /workspace/google-cloud-spanner-admin-database-v1/lib/google/cloud/spanner/admin/database/v1/database_admin/client.rb:1498:in `rescue in restore_database'
    /workspace/google-cloud-spanner-admin-database-v1/lib/google/cloud/spanner/admin/database/v1/database_admin/client.rb:1463:in `restore_database'
    /workspace/google-cloud-spanner/lib/google/cloud/spanner/service.rb:559:in `restore_database'
    /workspace/google-cloud-spanner/lib/google/cloud/spanner/backup.rb:358:in `restore'
    /workspace/google-cloud-spanner/acceptance/spanner/backup_test.rb:88:in `block (2 levels) in <top (required)>'
```

I don't have the permission to access Cloud Spanner on this project. Is there any way I can manually clean the restoring operation to unblock this PR? Many thanks!

**Review by hengfengli — APPROVED** [review]

**Comment by hengfengli** [comment]
@dazuma I thought we (cloud spanner team) are code owner for this directory, is our approval not enough? What is the current code review rule for a PR? Does it need two approvals from both cloud spanner team and yoshi-ruby team?

**Comment by hengfengli** [comment]
@skuruppu Are you able to merge this PR? Thanks.

**Review by skuruppu — APPROVED** [review]

**Comment by skuruppu** [comment]
No luck, I don't have permission :(

**Comment by hengfengli** [comment]
@JustinBeckwith Please merge this as well. Thanks.

**Pull request merged**
